gradle/any/dependencies.gradle

```diff
@@ -252,7 +258,7 @@ libraries["oro"] = "oro:oro:2.0.8"
 
 libraries["thymeleaf-spring4"] = "org.thymeleaf:thymeleaf-spring4:3.0.11.RELEASE"
 
-versions["edal"] = "1.4.2-SNAPSHOT"
```

Comment:
edal is fixed here

Reply:
<img width="1181" alt="image" src="https://user-images.githubusercontent.com/1993231/115062971-f2d2bb80-9eb8-11eb-86e9-479d280fe649.png">

Reply:
I remember having to change an edal-godiva lib, but if this is it, then we're good to go

Reply:
Please do try the build, I think it's building on jdk 1.8, but lemme double check...